### PR TITLE
Fix timestamp naming

### PR DESCRIPTION
**Pull request by dinogun**
Looks like the start_timestamp, end_timestamp -> interval_start_time, interval_end_time was missed out for two more classes

### CODE REVIEW

**Comment by dinogun** [comment]
@msvinaykumar please make sure this does not have implications from a DB point of view

**Review by bharathappali — APPROVED** [review]
LGTM!

File: src/main/java/com/autotune/common/data/result/ExperimentResultData.java [hunk 1]
```diff
@@ -31,9 +31,9 @@ public class ExperimentResultData {
     private String experiment_name;
     private String trialNumber;
     @SerializedName("interval_start_time")
```
Comment (khansaad) [inline src/main/java/com/autotune/common/data/result/ExperimentResultData.java:33]: I think this also can be changed to it's corresponding constant value similar to how it's done in the IntervalResults class

Comment (dinogun) [reply src/main/java/com/autotune/common/data/result/ExperimentResultData.java]: done, thanks

**Pull request merged**